src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -530,6 +524,18 @@ export const ShippingRoute: FC<ShippingProps> = props => {
     setShippingQuoteId(undefined)
   }
 
+  useEffect(() => {
+    if (
+      addressList &&
+      addressList.length > 0 &&
```

Comment:
**non-blocking**: you could achieve these 2 statements with `addressList?.length`